### ISSUE TITLE
Check which is the newest wrapper for multiple wrappers

### ISSUE THREAD

**Issue by paulzierep** [bebatut/galaxy_tool_extractor#17]
Thanks to @tuncK we found that there are tools that have been migrated (e.g. to IUC), example: https://github.com/bgruening/galaxytools/tree/master/tools/mcl
So they exist in multiple locations.
I think the tool_extractor overwrites an entry if the names are identical, so we would need two things:
a) allow storing multiple entries for tools with identical `Galaxy wrapper id` (maybe we should rename that to shed id as well  ?)
b) a way to filter for the newer wrapper, maybe based on the one with the latest commit.

**Comment by bebatut** [comment]
I do not think we should allow multiple entries for tools with identical wrapper ids. We should probably remove the duplicates in the different GitHub repositories
